redux-todomvc/containers/App.js

```diff
@@ -31,6 +31,7 @@ function mapStateToProps(state) {
 }
 
 function mapDispatchToProps(dispatch) {
+  backboneDispatch(dispatch, window.Backbone, window._);
```

Comment:
Pass in Backbone and underscore because your Marionette app may use globals or requirejs, etc. We're not going to judge.